src/server/worldserver/worldserver.conf.dist

```diff
@@ -3946,6 +3946,14 @@ Currency.StartJusticePoints = 0
 
 Currency.MaxJusticePoints = 4000
 
+#
+#    Currency.StartArtifactKnowledge
+#        Amount artifact knowledge that new players will start with
+#        Default: 55 (max)
+#
+
+Currency.StartArtifactKnowledge = 55
```

Comment:
is that retail behavoir?

Reply:
On retail Artifact Knowledge start at 40 and upgrade to 1 each week, but at this time it's 55 (max) so yes.

Reply:
Well, blizzard increased the start level with time. At release of the addon you started at 0.

Reply:
At start of legion yeah, but not since 7.3 release, you start with 40 of artifact knowledge and all players gain 1 rank each week. Source: http://www.wowhead.com/artifact-knowledge-guide